app/utils/statusHelper.py

```diff
@@ -0,0 +1,22 @@
+
+import Enum
+class Status(Enum):
+	UPLOADING = "Uploading video"
```

Comment:
I'm not liking the indents on this file.  Are you sure this is 4 spaces per tab?